docs/getting-started.md

```diff
@@ -119,85 +90,63 @@ We recommend choosing a directory outside of `$GOPATH` or `~/go`.
+make all
 ```
 
 If you clone repository inside the `$GOPATH` and golang version is 1.11.x, should explicitly set `GO111MODULE=on` before build.
 
 ```bash
-GO111MODULE=on make STATIC_AGENT=true
+GO111MODULE=on make all
 ```
 
-Once you have built the runtime, be sure to place the
-`containerd-shim-aws-firecracker` binary on your `$PATH`.
-
-### Inject the agent, runc shim, and runc into the rootfs image
-
-firecracker-containerd needs two components to be embedded in the root
-filesystem image.  They are:
-
-* `runc` - used to run containers
-* firecracker-containerd's `agent` - this is a process that communicates with
-  the outer firecracker-containerd `runtime` process over a `vsock` and proxies
-  commands to `containerd-shim-runc-v1`
-
-In order to inject these components in the root filesystem image above, we'll
-need to do a few steps.  These instructions assume that you're using the image
-described above; they are specific to the Alpine Linux user user-space and its
-init system.
-
-1. Increase the size of the root filesystem image.  The image that is supplied
-   by Firecracker (`hello-rootfs.ext4`) is an ext4 filesystem that is sized
-   exactly to the files already in it; no free space.  We can resize this
-   filesystem (while unmounted) by running
-   `e2fsck -f hello-rootfs.ext4 && resize2fs hello-rootfs.ext4`.
-2. Construct a `fc-agent.start` file so that the agent starts on boot:
-   <details>
-   <summary>fc-agent.start</summary>
-   
-   ```bash
-   #!/bin/sh
-   mkdir -p /container/rootfs
-   exec > /container/agent-debug.log # Debug logs from the agent
-   exec 2>&1
-   touch /container/runtime
-   mount -t auto -o rw /dev/vdb /container/rootfs
-   cd /container
-   /usr/local/bin/agent -id 1 -debug &
-   ```
-   </details>
-3. Mount the filesystem to a location like `/tmp/mnt`
-   <details>
-   <summary>Mounting at /tmp/mnt</summary>
-   
-   ```bash
-   sudo mkdir /tmp/mnt
-   sudo mount hello-rootfs.ext4 /tmp/mnt
-   ```
-   </details>
-4. Copy in the binaries, copy the `fc-agent.start` file, and set up OpenRC to
-   launch the necessary components
-   <details>
-   <summary>Copy in the binaries to /tmp/mnt</summary>
-   
-   ```bash
-   sudo cp $(which runc) firecracker-containerd/agent/agent /tmp/mnt/usr/local/bin
-   sudo cp fc-agent.start /tmp/mnt/etc/local.d
-   sudo chmod +x /tmp/mnt/etc/local.d/fc-agent.start
-   sudo ln -s /etc/init.d/local /tmp/mnt/etc/runlevels/default/local
-   sudo ln -s /etc/init.d/cgroups /tmp/mnt/etc/runlevels/default/cgroups
-   sudo umount /tmp/mnt
-   ```
-   </details>
-
-### Configure containerd snapshotter
-
-Add the snapshotter plugin to your `/etc/containerd/config.toml`
+Once you have built the runtime, be sure to place the following binaries on your
+`$PATH`:
+* `runtime/containerd-shim-aws-firecracker`
+* `snapshotter/cmd/devmapper/devmapper_snapshotter`
+* `snapshotter/cmd/naive/naive_snapshotter`
+* `firecracker-control/cmd/containerd/firecracker-containerd`
```

Comment:
or run `make install` ? We have `install` targets for all our binaries

Reply:
`make install` is probably a better idea.  I didn't know about `INSTALLROOT` and its behavior before looking today.

I can change this to `make install INSTALLROOT=/usr/local` or something similar.

Reply:
Done.